pkg/login/social/social.go

```diff
@@ -223,3 +238,35 @@ var GetOAuthProviders = func(cfg *setting.Cfg) map[string]bool {
 
 	return result
 }
+
+func readConfig(configFile string) (*setting.OAuthGroupMappingsConfig, error) {
+	if configFile == "" {
+		return nil, nil
+	}
+	result := &setting.OAuthGroupMappingsConfig{}
+
+	oauthLogger.Info("OAuth enabled, reading config file", "file", configFile)
+
+	fileBytes, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		return nil, errutil.Wrap("Failed to load OAuth config file", err)
+	}
+
+	_, err = toml.Decode(string(fileBytes), result)
```

Comment:
Think you can replace this with `_, err = toml.DecodeFile(configFile, result)`

Reply:
Change made